lib/internal/crypto/webcrypto.js

```diff
@@ -792,13 +792,6 @@ ObjectDefineProperties(
       writable: true,
       value: randomUUID,
     },
-    CryptoKey: {
-      __proto__: null,
-      enumerable: true,
-      configurable: true,
-      writable: true,
-      value: CryptoKey,
-    }
```

Comment:
I believe this should stay unless global is enabled as the only way of obtaining the constructor for instanceof use.

Reply:
Yeah, good catch, I didn't think about that. Maybe we should simply expose it on `require('node:crypto')`?

Reply:
> Maybe we should simply expose it on require('node:crypto')?

I would not like introducing yet another variation to this.